exceptions.rst

```diff
@@ -100,7 +103,10 @@ example:
     except Exception:
         print('exception')
     else:
-        print('This would only run if no exception occurs.')
+        # any code that should only run if no exception occurs in the try,
```

Comment:
and I added a comment in the code and print string explaining it again in the context of code